src/main/kotlin/org/stellar/walletsdk/utils/buildFeeBumpTransaction.kt

```diff
@@ -0,0 +1,24 @@
+package org.stellar.walletsdk.utils
+
+import java.io.IOException
+import org.stellar.sdk.FeeBumpTransaction
+import org.stellar.sdk.Server
+import org.stellar.sdk.Transaction
+
+fun buildFeeBumpTransaction(
+  feeAccount: String,
+  innerTransaction: Transaction,
+  maxBaseFeeInStroops: Long,
+  server: Server
+): FeeBumpTransaction {
+  try {
+    server.accounts().account(feeAccount)
+  } catch (e: IOException) {
+    throw Exception("Fee account was not found")
+  }
```

Comment:
I remember this repeats a couple of times. It may make sense to have a function such as `validateAddress()` in the project.